docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-post-kcp.md

```diff
@@ -92,11 +92,66 @@ to pass the new file with `--kubeconfig` on the command lines when
 using kcp or KubeStellar.
 
 
+### Space Manager
+
+The Space Manager will use space provider to create and manage underlying cluster
+for each Space object. Space objects are created in `root:space-mgt` context
+under `spaceprovider-default` namespace.
+
+```shell
+space-manager --context root:space-mgt &
```

Comment:
Since `kubestellar init` is doing this, why is it also being done separately?

Reply:
On `kubestellar init` we only create space-mgt workspace and a context for this workspace. The `space-manager` controller is started on `kubestellar start`. Currently no controllers are started on init. Should we start `space-manager` on init?